README.md

```diff
@@ -1,2 +1,62 @@
-# educodar.github.io
-Live
+# Educodar
+
+## Como utilizar este repositório
+
+### 1. Clone este repositório em sua máquina
+
+Você precisa ter o Git instalado em seu Windows, macOS ou Linux.
+Então, execute o seguinte comando no terminal de sua máquina:
+
+```bash
```

Comment:
No Windows é PowerShell (ou Command Prompt) e tem diferenças com bash.

Reply:
Um post do StackOverflow recomenda, na verdade, que se use `shell` em vez de `bash` nessa situação. Vou criar uma issue de débito técnico e colocar na Icebox para trocarmos esse conteúdo no README.md depois

Reply:
@LucasCarvalhais I changed `bash` to `shell` on https://github.com/educodar/website/commit/a50e561351adf7d1513f86a3ff167d72e4171f99#diff-04c6e90faac2675aa89e2176d2eec7d8